BlackJack/index.js

```diff
@@ -29,189 +41,295 @@ makeCard();
 
 //onpage card
 let dealerRandomCard = Math.floor(Math.random() * allCard.length);
-dealerImg.src = `cards/${allCard[dealerRandomCard]}`
-dealerCards.appendChild(dealerImg)
-let dealerCardN = allCard[dealerRandomCard]
-let dealerCardNum = dealerCardN.charAt(0)
+dealerImg.src = `cards/${allCard[dealerRandomCard]}`;
+dealerCards.appendChild(dealerImg);
+let dealerCardN = allCard[dealerRandomCard];
+let dealerCardNum = dealerCardN.charAt(0);
 
 //dealer hidden card
 // let dealerHiddenRandomCard = Math.floor(Math.random() * allCard.length) +1;
 
-
 let playerRandomCard = Math.floor(Math.random() * allCard.length);
-playerImg.src = `cards/${allCard[playerRandomCard]}`
-playerCards.appendChild(playerImg)
-
-let playerRandomCard2 = Math.floor(Math.random() * allCard.length) +1;
-playerImg2.src = `cards/${allCard[playerRandomCard2]}`
-playerCards.appendChild(playerImg2)
+playerImg.src = `cards/${allCard[playerRandomCard]}`;
+playerCards.appendChild(playerImg);
 
+let playerRandomCard2 = Math.floor(Math.random() * allCard.length) + 1;
+playerImg2.src = `cards/${allCard[playerRandomCard2]}`;
+playerCards.appendChild(playerImg2);
```

Comment:
I suppose what you wanted to do was to make sure the card wont get repeated? 
+1 on the index will not work because now it will try to get the from card array between index number 1 and 52. Which is not removing the previous card, but always removing the card at index 0. And also, index 52 will cause the card to be undefined.